pkg/transformers/greenmask/greenmask_integer_transformer_test.go

```diff
@@ -0,0 +1,415 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package greenmask
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/xataio/pgstream/pkg/transformers"
+)
+
+func TestNewIntegerTransformer(t *testing.T) {
```

Comment:
In most cases, unit tests can be run in parallel, since they don't affect each other. This helps speeding up the test runs. 
You just need to add a `t.Parallel()` both here and below in the for loop, so that this test, and all the subcases are run concurrently. 
You can see the examples in the other test files:
https://github.com/xataio/pgstream/blob/main/pkg/transformers/greenmask/greenmask_string_transformer_test.go#L13
https://github.com/xataio/pgstream/blob/main/pkg/transformers/greenmask/greenmask_string_transformer_test.go#L82